x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -354,6 +360,7 @@ const FormattedFieldValueComponent: React.FC<{
             ? null
             : fieldName
         }
+        params={{ isEmptyString }}
```

Comment:
@michaelolo24 so this is where its different from the last take - hover actions can now extract this from the `dataProvider`